src/interpreter/ByteCodeInterpreter.cpp

```diff
@@ -710,7 +711,11 @@ Value ByteCodeInterpreter::interpret(ExecutionState* state, ByteCodeBlock* byteC
             BinaryBitwiseAnd* code = (BinaryBitwiseAnd*)programCounter;
             const Value& left = registerFile[code->m_srcIndex0];
             const Value& right = registerFile[code->m_srcIndex1];
-            registerFile[code->m_dstIndex] = Value(left.toInt32(*state) & right.toInt32(*state));
+            if (left.isInt32() && right.isInt32()) {
```

Comment:
What about adding "LIKELY" macro to this likely-happen condition?
It could be applied to other bitwise and shift operations within interpreter too.

Reply:
If I added to LIKELY marco into that expression,
the benchmark score is down... :(
there are so many !isInt32 cases in real world

Reply:
I got it. Thanks :)